.github/workflows/triage.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow is built to manage the triage support by using GH issues.
+name: '[Support] Organize triage'
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+  pull_request_target:
+    types:
+      - reopened
+      - opened
+permissions:
+  repository-projects: write
```

Comment:
I think this is not needed (there is no projects in this repository)

Reply:
Fully agree @fmulero , good catch!